pkg/mount/block/blockdev.go

```diff
@@ -173,25 +160,19 @@ func GetBlockStats() ([]BlockDev, error) {
 		devnames = append(devnames, rel)
 		return nil
 	})
-	if err != nil {
-		return nil, err
-	}
+
+	var blockdevs []*BlockDev
 	for _, devname := range devnames {
-		buf, err := ioutil.ReadFile(fmt.Sprintf("%s/%s/stat", root, devname))
-		if err != nil {
-			return nil, err
-		}
-		bstat, err := BlockStatFromBytes(buf)
-		if err != nil {
-			return nil, err
-		}
-		devpath := path.Join("/dev/", devname)
+		devpath := filepath.Join("/dev/", devname)
 		if uuid, err := getUUID(devpath); err != nil {
-			blockdevs = append(blockdevs, BlockDev{Name: devname, Stat: *bstat})
+			blockdevs = append(blockdevs, &BlockDev{Name: devname})
 		} else {
-			blockdevs = append(blockdevs, BlockDev{Name: devname, Stat: *bstat, FsUUID: uuid})
+			blockdevs = append(blockdevs, &BlockDev{Name: devname, FsUUID: uuid})
 		}
 	}
+	if err != nil {
```

Comment:
Can you comment while this if-statement doesn't come immediately after `filepath.Walk`?

Reply:
I can't, it's just plain wrong

Reply:
moved this.